app/src/main/java/org/fossasia/openevent/general/MainActivity.kt

```diff
@@ -121,7 +121,6 @@ class MainActivity : AppCompatActivity() {
     private fun loadFragment(fragment: Fragment) {
         supportFragmentManager.beginTransaction()
                 .replace(R.id.frameContainer, fragment, fragment::class.java.name)
-                .addToBackStack(null)
```

Comment:
This is required to save fragments and their data between switching tabs and other things

Reply:
Used finish() so that the app closes when back is pressed while on the Events fragment. Is it okay now?